File: C program/combination.c
#include<stdio.h>
int main()
{
    int n,i,j,k;
    
    printf("Enter the value of n:\n");
    scanf("%d",&n);

    printf("The combination of digit is as follows:\n");
    for(i=0;i<n;i++)
    {
        for(j=0;j<n;j++)
        {
            for(k=0;k<n;k++)
            {
                if(i!=j && i!=k && j!=k)
                {
                    printf("%d%d%d",i,j,k);
                    printf("\n");
                }
            }
        }
    }
    printf("MISSION SUCCESSFUL");
}


/*
----------output----------
Enter the value of n:
3
The combination of digit is as follows:
012
021
102
120
201
210
MISSION SUCCESSFUL
*/

File: Java Programming/combination.java
public class combination
{
    public static void main(String[] args)
    {
        for(int i=0; i<4; i++)
        {
            for(int j=0; j<4; j++)
            {
                for(int k=0; k<4; k++)
                {
                    if(i !=j && j != k && i !=k)
                    {
                        System.out.println(i + " " + j + " " + k);
                    }
                }
            }
        }
    }
}





/*
--------------------0utput--------------------
0 1 2
0 1 3
0 2 1
0 2 3
0 3 1
0 3 2
1 0 2
1 0 3
1 2 0
1 2 3
1 3 0
1 3 2
2 0 1
2 0 3
2 1 0
2 1 3
2 3 0
2 3 1
3 0 1
3 0 2
3 1 0
3 1 2
3 2 0
3 2 1
*/
